com.unity.netcode.gameobjects/Runtime/Transports/UTP/UnityTransport.cs

```diff
@@ -1432,6 +1477,60 @@ private void ConfigureSimulatorForUtp1()
 #endif
                 heartbeatTimeoutMS: transport.m_HeartbeatTimeoutMS);
 
+#if UNITY_WEBGL && !UNITY_EDITOR
+            if (NetworkManager.IsServer)
+            {
+                throw new Exception("WebGL as a server is not supported by Unity Transport, outside the Editor.");
+            }
+#endif
+
+#if UTP_TRANSPORT_2_0_ABOVE
+            if (m_UseEncryption)
+            {
+                if (m_ProtocolType == ProtocolType.RelayUnityTransport)
+                {
+                    if (m_RelayServerData.IsSecure != 0)
+                    {
+                        // log an error because we have mismatched configuration
+                        Debug.LogError("Mismatched security configuration, between Relay and local NetworkManager settings");
+                    }
+                    else
+                    {
+                        // Todo: new code to support Relay+WSS
+                        throw new NotImplementedException();
```

Comment:
I'd just leave it empty instead of throwing an exception since secure Relay communications work correctly if using DTLS. Or maybe only throw the exception if using WebSockets, since that's the scenario that's not implemented.

Reply:
04a1d5e6